main.py

```diff
@@ -6,7 +6,8 @@
 
 from covid.constants import PATH_TO_SERVICE_ACCOUNT_KEY
 from covid.extract import extract_cdc_ili_data
-from covid.extract import extract_covidtracking_historical_data
+from covid.extract import extract_covidtracking_historical_data, extract_covidtracking_current_data
```

Comment:
Would you mind install pre-commit and running it according to the instructions i just added? https://github.com/usdigitalresponse/covid-exit-strategy/pull/42/files. Not your fault, this wasn't documented yet! But it'll move these to single-line imports.